specs/master/cluster-init/scripts/020_install_cyclecloud_hostfactory.sh

```diff
@@ -60,14 +56,14 @@ set -e
 #             j["allocated_hosts"] = []' ${EGO_TOP}/${EGO_VERSION}/hostfactory/requestors/symA/scripts/Main.py
 
 
-# echo "Starting HostFactory..."
-# sudo -i -u egoadmin bash << EOF
-# . /etc/profile.d/symphony.sh
-# egosh user logon -u ${SOAM_USER} -x ${SOAM_PASSWORD}
-# egosh service stop HostFactory
-# egosh service start HostFactory
-# egosh service view HostFactory
-# EOF
+echo "Starting HostFactory..."
+sudo -i -u egoadmin bash << EOF
+. /etc/profile.d/symphony.sh
+egosh user logon -u ${SOAM_USER} -x ${SOAM_PASSWORD}
+egosh service stop HostFactory
+egosh service start HostFactory
```

Comment:
We need to wait until HF has stopped here - I have definitely hit issues where I tried to start HF while it was still in deallocating state.

Reply:
yeah good point I have noticed this too, will have to view Hotfactory and check output until its stopped